### PR TITLE
[Docs] how to train a large model

### PR DESCRIPTION
**Pull request by zhouzaida**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/common_usage/large_model_training.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    model=MMResNet50(),
+    work_dir='./work_dirs',
+    strategy=strategy,
+    train_dataloader=train_dataloader,
+    optim_wrapper=optim_wrapper,
+    param_scheduler=dict(type='LinearLR'),
+    train_cfg=dict(by_epoch=True, max_epochs=10, val_interval=1),
+    val_dataloader=val_dataloader,
+    val_cfg=dict(),
+    val_evaluator=dict(type=[Accuracy, MMResNet50]))
```
Comment (HAOCHENYE) [inline docs/zh_cn/common_usage/large_model_training.md:74]: Why `MMResNet50` is passed to `val_evaluator` ?

**Review by HAOCHENYE — DISMISSED** [review]

File: docs/zh_cn/common_usage/large_model_training.md (hunk 1 above)
Comment (zhouzaida) [inline docs/zh_cn/common_usage/large_model_training.md:74]:
```suggestion
    val_evaluator=dict(type=Accuracy))
```

File: docs/zh_cn/common_usage/large_model_training.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
```
Comment (zhouzaida) [inline docs/zh_cn/common_usage/large_model_training.md:146]:
```suggestion
    val_evaluator=dict(type=Accuracy))
```

File: docs/en/common_usage/large_model_training.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    model=MMResNet50(),
+    work_dir='./work_dirs',
+    strategy=strategy,
+    train_dataloader=train_dataloader,
+    optim_wrapper=optim_wrapper,
+    param_scheduler=dict(type='LinearLR'),
+    train_cfg=dict(by_epoch=True, max_epochs=10, val_interval=1),
+    val_dataloader=val_dataloader,
+    val_cfg=dict(),
+    val_evaluator=dict(type=[Accuracy, MMResNet50]))
```
Comment (zhouzaida) [inline docs/en/common_usage/large_model_training.md:75]:
```suggestion
    val_evaluator=dict(type=Accuracy))
```

File: docs/en/common_usage/large_model_training.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
```
Comment (zhouzaida) [inline docs/en/common_usage/large_model_training.md:147]:
```suggestion
    val_evaluator=dict(type=Accuracy))
```

**Pull request merged**